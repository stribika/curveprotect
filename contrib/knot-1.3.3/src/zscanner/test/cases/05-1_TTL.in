$ORIGIN	.

; KO
$TTL	4294967296	; 32bit overflow
