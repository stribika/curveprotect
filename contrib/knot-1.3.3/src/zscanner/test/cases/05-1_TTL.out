ERROR=ZSCANNER_ENUMBER32_OVERFLOW
------
